app/views/games/finished.html.erb

```diff
@@ -9,7 +9,7 @@
       </div>
     </div>
     <div class="too_late">
-      <% if @game.players.include?(current_player) && current_player.won == false %>
+      <% if @game.players.include?(current_player) && current_player.won %>
```

Comment:
Regarding the `#loser_is?`method, we need to know who is watching the page:
- the winner
- the loser
- any other player (signed in or not)

Or we can just have `#winner` / `#loser` and check `if game.winner == current_player` (less nice).
What do you think?